source/templates.tex

```diff
@@ -53,7 +53,7 @@
 The \tcode{>} token following the
 \grammarterm{template-parameter-list} of a
 \grammarterm{template-declaration}
-may be the product of replacing a
+could be the product of replacing a
```

Comment:
```suggestion
can be the product of replacing a
```
"can" seems no worse than "could" here and is more ISO-compatible.

Reply:
fixed